torch/csrc/api/src/nn/init.cpp

```diff
@@ -108,12 +105,12 @@ Tensor eye_(Tensor matrix) {
   return torch::eye_out(matrix, matrix.size(0), matrix.size(1));
 }
 
-Tensor normal_(Tensor tensor, double mean, double std) {
+Tensor normal_(const Tensor& tensor, double mean, double std) {
   NoGradGuard guard;
   return tensor.normal_(mean, std);
 }
```

Comment:
It's all kind of pointless since the APIs here return a Tensor so you have to increment the refcount anyway one way or another

Reply:
But the underlying interfaces accept const &, so at least we can reduce one increment?

Reply:
Reverted